### PR TITLE
don't print creating when in mock mode

### PR DESCRIPTION
**Pull request by mphammer**
QA said it was confusing that synopsysclt stated it was creating the resource into a namespace when running in mock mode.

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/517#" title="Author self-approved">mphammer</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [mphammer]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@mphammer: PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: pkg/synopsysctl/cmd_update.go [hunk 1]
```diff
@@ -570,14 +580,23 @@ var updateBlackDuckAddEnvironCmd = &cobra.Command{
 			return fmt.Errorf("cannot update Black Duck '%s' in namespace '%s' due to %+v", blackDuckName, blackDuckNamespace, err)
 		}
 		if canUpdate {
+			// Create new Black Duck CRD
+			newBlackDuck := *currBlackDuck //make copy
```
Comment (msenmurugan) [inline pkg/synopsysctl/cmd_update.go]: @mphammer is it required to make a copy?

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/517#" title="Author self-approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/517#pullrequestreview-252996972" title="LGTM">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [mphammer,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@rrati: #517 failed to apply on top of branch "release-2019.6.x":
```Applying: don't print creating when in mock mode
error: Failed to merge in the changes.
Using index info to reconstruct a base tree...
M	pkg/synopsysctl/cmd_create.go
M	pkg/synopsysctl/cmd_update.go
Falling back to patching base and 3-way merge...
Auto-merging pkg/synopsysctl/cmd_update.go
CONFLICT (content): Merge conflict in pkg/synopsysctl/cmd_update.go
Auto-merging pkg/synopsysctl/cmd_create.go
Patch failed at 0001 don't print creating when in mock mode

```

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/517#issuecomment-504550597):

>/cherrypick release-2019.6.x


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**